File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        //first set create pananum
        Set<Integer> set = new HashSet<>();

        //add the all elements into set

        for(int n:nums)
        set.add(n);

        int max = 0;
        for(int i=0; i<nums.length; i++)
        {
            int num = nums[i];
            if(set.contains(num - 1)) continue;

            int current_len = 0;
            while(set.contains(num))
            {
                current_len+=1;
                num+=1;
            }

            max = Math.max(max,current_len);
        }

        return max;
    }
}

File: Data Structures & Algorithms/majority-element/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> int:
        map = {}
        for n in nums:
            map[n] = map.get(n,0)+1
        
        for key,val in map.items():
            if val > len(nums) // 2:
                return key

 